libs/examples/src/indicator/indicator-default-example/indicator-default-example.html

```diff
@@ -0,0 +1,80 @@
+<dt-tree-table [dataSource]="dataSource" [treeControl]="treeControl">
```

Comment:
I think we should scale down the indicator example to a simple span with a dtIndicator on it and a button to change the color from error to warning. We can reference the more complex example for the tree table in the readme which will showup on barista. Even a link is ok for me there